Introduce non-interactive mode (#35)
ansys/pysimai: doc/source/user_guide/configuration.rst

To use the SimAI API, your :class:`~ansys.simai.core.client.SimAIClient`
instance must be authenticated. By default, you are prompted to log in
via your web browser. However, you can pass your credentials as parameters

> on client creation in the interactive mode (see `Interactive mode`_.):

Reviewer: You can also set them in both modes. So I'd just remove that mention or mention it's required in non-interactive mode.

Author: True, I misread it.

.. autopydantic_model:: Credentials
  :model-show-config-summary: False
  :model-show-validator-summary: False
  :model-show-validator-members: False
  :model-show-json: False
  :field-list-validators: False

.. _Interactive mode:

Interactive mode
------------------

> When the property `interactive` is set to `true`, the users could enter configuration
> properties from the terminal as they get prompted.

Reviewer: ```suggestion
When the property `interactive` is set to `true`, the user will be prompted for missing configuration options.
```

Author: I updated this part slightly different from this suggestion but along the same lines.

When the property is `false`, the interactive mode is disabled, and errors would be raised
in case of missing configuration properties.
Default behavior is `interactive=true`.

> It is important to note that credentials become required when `interactive=false`.

Reviewer: ```suggestion
It is important to note that login through web browser is disabled and credentials become required when `interactive=false`.
```
